mininet/opticalTest.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+
+from opticalUtils2 import MininetOE, LINCSwitch, LINCLink
```

Comment:
yes, its opticalTest.py. let me fix that

Thanks,
Fahad.
On Feb 24, 2015, at 12:59 PM, Jon Hall <notifications@github.com> wrote:

> In mininet/opticalTest.py:
> 
> > @@ -0,0 +1,93 @@
> > +#!/usr/bin/python
> > +
> > +from opticalUtils2 import MininetOE, LINCSwitch, LINCLink
> is opticalUtils2 a typo?
> 
> —
> Reply to this email directly or view it on GitHub.
>